src/index.ts

```diff
@@ -1,2 +1,14 @@
-console.log("Hello world! Usman");
+import 'reflect-metadata';
+import http from 'http';
+import { setupExpressServer } from "./adapters/express";
+import {inversifyContainer} from './containers/inversify.container'
+import { LoggerService } from './services/logger.service';
 
+const expressApplication = setupExpressServer();
```

Comment:
I think this should be just a setupServer() instead of Express Server. For example, if we just use another framework we will just change it in the adapter pattern not here.

Reply:
in case there are other servers from a different framework then express how would we know?